### ISSUE TITLE
Download file missing activities

### ISSUE THREAD

**Issue by temi** [AtlasOfLivingAustralia/biocollect#1428]
This bug is reported by user - https://support.ehelp.edu.au/a/tickets/149960.
User executed a query to retun 5 activities. But download only returned 4 activities. Downloading missing record retuned an empty file.

Project - [Bulimba Creek Catchment nest box monitoring project](https://biocollect-staging.ala.org.au/acsa/project/index/efde3b5d-fc96-4c72-88b4-c8ebb1627166)
Screenshot - 
[Screen shot 5 records.pdf](https://github.com/AtlasOfLivingAustralia/biocollect/files/8937243/Screen.shot.5.records.pdf)

FQ - 
```
fq: projectActivityNameFacet:Bulimba%20Creek%20NB%20Project%20-%20Location%20Survey
fq: siteNameFacet:IP%20NB01
fq: siteNameFacet:IP%20NB02
fq: siteNameFacet:IP%20NB03
fq: siteNameFacet:IP%20NB04
fq: siteNameFacet:IP%20NB05
```

**Comment by chamaliperera512** [comment]
This issue was tried to be recreated with a latest staging DB dump using different ecodata branches.

When the download was executed with:
1. ecodata dev branch - this issue was able to be recreated
2. ecodata hotfix/3.8.2 (which contains the fixes for download related issues) - this issue was NOT able to be recreated

Hence the download related fixes in hotfix/3.8.2 have rectified this issue.

**Comment by temi** [comment]
passed
